packages/js-sdk/src/sandbox/filesystem/index.ts

```diff
@@ -21,7 +15,8 @@ import { EnvdApiClient } from '../../envd/api'
 import { Filesystem as FilesystemService } from '../../envd/filesystem/filesystem_connect'
 import { FileType as FsFileType } from '../../envd/filesystem/filesystem_pb'
 
-import { WatchHandle, FilesystemEvent } from './watchHandle'
+import { clearTimeout } from 'timers'
```

Comment:
The clearTimeout should be a built in JS method that does not need to be imported.

Reply:
ah snap, this is my import sorter extension doing this